sorc/link_fv3gfs.sh

```diff
@@ -180,12 +183,12 @@ if [ $machine = dell -o $machine = hera ]; then
     done
 fi
 
-for ufs_utilsexe in chgres_cube nemsio_get nemsio_read global_chgres ; do
+for ufs_utilsexe in chgres_cube ; do # nemsio_get nemsio_read global_chgres ; do
```

Comment:
Remove old executables

Reply:
No problem. I will delete in the next commit.